packages/preview-middleware-client/src/adp/init.ts

```diff
@@ -34,6 +39,8 @@ export default async function (rta: RuntimeAuthoring) {
     const applicationType = getApplicationType(rta.getRootControlInstance().getManifest());
     const quickActionRegistries = await loadDefinitions(applicationType);
 
+    
+
```

Comment:
Empty rows, some of the files in the client folder are not prettier'd, could you please check?